### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by ekoby**
fixed some naming leftovers and a few warnings

### CODE REVIEW

File: include/tlsuv/src_t.h [hunk 1]
```diff
@@ -13,23 +13,20 @@
 // limitations under the License.
 
 /**
- * @file um_http_src_t.h
- * @brief header file for um_http_src_t type
+ * @file ttp_src_t.h
```
Comment (scareything) [inline include/tlsuv/src_t.h:16]: should be `src_t.h`?

File: include/tlsuv/src_t.h [hunk 2]
```diff
@@ -13,23 +13,20 @@
...
+ * @brief header file for tlsuv_src_t type
```
Comment (scareything) [inline include/tlsuv/src_t.h:17]: the typedef now removed, so should this be `struct tlsuv_src_s`?

File: src/mbedtls/engine.c
Comment (scareything) [inline src/mbedtls/engine.c:1]: do the `c->own_key` warnings make sense to you?

https://github.com/openziti/tlsuv/actions/runs/4483060794/jobs/7882229851?pr=139#step:7:418

File: src/openssl/engine.c
Comment (scareything) [inline src/openssl/engine.c:1]: more `own_key` pointer type warnings here

**Review by scareything — APPROVED** [review]

File: include/tlsuv/src_t.h (hunk 2 above)
Comment (ekoby) [reply include/tlsuv/src_t.h:17]: the type is still there, the warning was because it `typdef`ed in both places

File: src/openssl/engine.c
Comment (ekoby) [reply src/openssl/engine.c:1]: yep, fixed

**Pull request merged**